flixel/FlxCamera.hx

```diff
@@ -120,30 +120,30 @@ class FlxCamera extends FlxBasic
 	 */
 	public var scroll:FlxPoint;
 	
-	#if FLX_RENDER_BLIT
-	/**
-	 * The actual bitmap data of the camera display itself.
-	 */
-	public var buffer:BitmapData;
-	/**
-	 * Whether checkResize checks if the camera dimensions have changed to update the buffer dimensions.
-	 */
-	public var regen:Bool = false;
-	#end
+	//start renderBlit
```

Comment:
If you're gonna add a comment for these "regions" (which I'm not sure is a good idea as I doubt it'll be maintained very well), you might as well take advantage of [FD's regions](http://stackoverflow.com/a/2617026/2631715) for code folding etc.